client/transaction-pool/src/lib.rs

```diff
@@ -437,7 +441,14 @@ impl<PoolApi, Block> MaintainedTransactionPool for BasicPool<PoolApi, Block>
 
 							resubmit_transactions.extend(block_transactions);
 						}
-						if let Err(e) = pool.submit_at(&id, resubmit_transactions, true).await {
+						if let Err(e) = pool.submit_at(
+							&id,
+							// These transactions are coming from retracted blocks, we should
```

Comment:
what is a retracted block?

Reply:
A block that was consider canonical, but due to re-organisation is not any more.
We re-import transactions from such blocks back to the pool.